adafruit_ntp.py

```diff
@@ -48,15 +48,17 @@ class NTP:
     This module does not handle daylight savings or local time.
 
     :param adafruit_esp32spi esp: ESP32SPI object.
+    :param bool debug: Set to True to output set_time() failures to console
     """
 
-    def __init__(self, esp):
+    def __init__(self, esp, debug: bool = False):
```

Comment:
This can simply be `debug=False`.

```suggestion
    def __init__(self, esp, debug=False):
```